src/main/java/com/SAS/soccer_association_system/GamesAPIController.java

```diff
@@ -37,6 +37,11 @@ public String getGameEvents(@PathVariable String gameID) {
         return GameCRUD.getGameEvents(gameID).toString();
     }
 
+    @GetMapping(value = "/referee/{gameID}")
+    public String getRefereeForGame(@PathVariable String gameID) {
+        return GameCRUD.getRefereesName(gameID).toString();
```

Comment:
It suppose to call a function in the sasApplication and from there call the gameController and from there to the GameCRUD.

Reply:
added a TBD